.rubocop.yml

```diff
@@ -59,6 +64,18 @@ Style/BlockDelimiters:
   Enabled: true
   EnforcedStyle: line_count_based
 
+Style/SignalException:
+  EnforcedStyle: semantic
+
+Style/TrailingCommaInLiteral:
+  EnforcedStyleForMultiline: consistent_comma
```

Comment:
Is the trailing comma what all the cool kids do these days? https://github.com/rails-api/active_model_serializers/pull/1560/files#diff-57dc91570125bb0a036a83a5c8bd6be8R63

Usually I'm fine with it being allowed, not necessarily required, as is a habit I picked up from Python for this. I'm curious if others have any strong opinion here.